services/generateAnonUserInfo/generateAnonUserInfoServices.ts

```diff
@@ -0,0 +1,27 @@
+import { AxiosResponse } from "axios";
+import apiAnonymous from "@shared/fetcherAnonymous";
+
+type GenerateAnonUserInfoResponse = {
+  targetUserId: string;
+  myAnonUserId: string;
+  anon_user_info_emoji_name: string;
+  anon_user_info_emoji_code: string;
+  anon_user_info_color_name: string;
+  anon_user_info_color_code: string;
+};
+
+type GenerateAnonUserInfoPayload = {
+  userId: string;
+};
+
+const generateAnonUserInfo = async (
+  payload: GenerateAnonUserInfoPayload
+): Promise<GenerateAnonUserInfoResponse> => {
+  const { data } = await apiAnonymous.post<
+    AxiosResponse<GenerateAnonUserInfoResponse>
+  >(`/api/v1/chat/users/${payload.userId}`, {});
+  return data.data;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `generateAnonUserInfo` is making a POST request to an API endpoint that seems to be related to chat users. However, it's not clear why a POST request is being used here as the function name and the payload suggest that this function is intended to retrieve (not create or update) anonymous user information. If this function is indeed meant to retrieve data, a GET request would be more appropriate.

If the POST request is necessary and you are creating or updating data on the server, consider renaming the function to reflect this. Also, ensure that any necessary data is included in the body of the POST request. Currently, an empty object `{}` is being sent which might not be correct.

```diff
-  const { data } = await apiAnonymous.post<
-    AxiosResponse<GenerateAnonUserInfoResponse>
-  >(`/api/v1/chat/users/${payload.userId}`, {});
+  const { data } = await apiAnonymous.get<
+    AxiosResponse<GenerateAnonUserInfoResponse>
+  >(`/api/v1/chat/users/${payload.userId}`);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->